### PR TITLE
Rtdt integration

### PR DESCRIPTION
**Pull request by aswinjayan94**
Integrate RT/DT code into the master branch. It has been tested and found to be working well (check s3 repository for results)

The code is scheduled to run 1 hour after the 'Update Covid Dashboard' workflow (at 2300hrs UTC)

### CODE REVIEW

File: .github/workflows/Run_rt_calcs.yml [hunk 1]
```diff
@@ -3,84 +3,83 @@ name: R_proc
...
+      - name: Configure AWS Credentials for data repository
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ap-south-1
+          
+      - name: Executing Wards UpdateEpiStatsTask luigi task
+        run: |
+          python -m luigi --module tasks UpdateEpiStatsWardsTask --local-scheduler
```
Comment (rivera-lanasm) [inline .github/workflows/Run_rt_calcs.yml:81]: this looks good I think, running both the district and ward RT/DT processes here

File: tasks/epi_stats/fetch_citystats.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    
+    
+    n = 20  #################### TO n, ASSIGN NUMBER OF CITIES SATISFYING CRITERIA 1 TO BE PICKED #########
+    criteria1 = (c1a & c1b)
+    critical_cities_c1 = data_latest[criteria1][['district', 'delta.confirmed']]
+    critical_cities_c1.sort_values(by=['delta.confirmed'], ascending=False, inplace=True)
+    critical_cities_c1_capped = critical_cities_c1.head(n)    
+    
+    # criteria 2
+    top20_cutoff = data_latest["total.confirmed"].sort_values(ascending=False).iloc[20]
```
Comment (rivera-lanasm) [inline tasks/epi_stats/fetch_citystats.py:39]: is that 20 supposed to be the "n" variable

**Review by rivera-lanasm — APPROVED** [review]
these updates look good. Thanks for your help Aswin!

File: tasks/epi_stats/fetch_citystats.py (hunk 2 above)
Comment (aswinjayan94) [reply tasks/epi_stats/fetch_citystats.py:39]: No the 20 is for criteria 2 (top 20 by cumulative) and n is for criteria 1 (top n by delta.confirmed of those districts satisfying criteria 1) are supposed to be different (i understand the confusion)

**Pull request merged**